src/server/models/education.model.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+let educationSchema = new Schema({
+    schoolName: String,
+    degree: String,
+    state: String,
```

Comment:
Why is state important? Let's remove it for now and later define schoolName as another school object that has more info about the school.